packages/fxa-settings/src/pages/ResetPassword/index.tsx

```diff
@@ -93,59 +95,67 @@ const ResetPassword = ({
     }
   }, [errorText, setErrorText]);
 
-  const localizedEmailRequiredMessage = ftlMsgResolver.getMsg(
-    'reset-password-email-required-error',
-    'Email required'
+  const submitEmail = useCallback(
+    async (email: string) => {
+      try {
+        clearError();
+        const result = await account.resetPassword(email);
+        navigateToConfirmPwReset({
+          passwordForgotToken: result.passwordForgotToken,
+          email,
+        });
+      } catch (err) {
+        let localizedError;
+        if (err.errno && AuthUiErrorNos[err.errno]) {
+          if (
+            err.errno === AuthUiErrors.THROTTLED.errno &&
+            err.retryAfterLocalized
+          ) {
+            localizedError = ftlMsgResolver.getMsg(
+              composeAuthUiErrorTranslationId(err),
+              AuthUiErrorNos[err.errno].message,
+              { retryAfter: err.retryAfterLocalized }
+            );
+          } else {
+            localizedError = ftlMsgResolver.getMsg(
+              composeAuthUiErrorTranslationId(err),
+              AuthUiErrorNos[err.errno].message
+            );
+          }
+        } else {
+          // TEMPORARY deliberate log to help debug FXA-7347, this should be captured server-side
+          // but for some reason isn't logging to Sentry
+          sentryMetrics.captureException(err);
+          const unexpectedError = AuthUiErrors.UNEXPECTED_ERROR;
```

Comment:
This was my previous work here - but not sure if we're okay with assigning `AuthUiErrors.UNEXPECTED_ERROR` to errors that might not have an error number? Wondering if that muddles the waters for figuring out what the problem is in FXA-7347

Reply:
I'm working on that Settings architecture doc this next sprint and I want to leave some notes around our error handling and messaging anyway so I'll circle back here.

I don't think it's muddying the waters though since AFAICT this just serves as a catch-all to display a generic error message in the UI if the err doesn't match any of the previous cases - this would not affect the response object from gql-api, which is the source (or rather this or auth-server) of whatever's going on.